### PR TITLE
Checking also for "ember build --prod" command.

### PR DESCRIPTION
**Pull request by davideferre**
Added condition to check also `ember build --prod` command.

### CODE REVIEW

**Comment by jasonmit** [comment]
This seems like a good stop gap until we're able to remove this function when the upstream issues are resolved.  I'll cut a patch release tonight.

File: index.js [hunk 1]
```diff
@@ -54,8 +54,8 @@ module.exports = {
     let args = parseArgs(process.argv);
     let env = args.e || args.env || args.environment;
 
-    // Is it "ember b -prod" command?
-    if (!env && process.argv.indexOf('-prod') > -1) {
+    // Is it "ember b -prod" or "ember build --prod" command?
+    if (!env && ((process.argv.indexOf('-prod') > -1) || (process.argv.indexOf('--prod') > -1))) {
```
Comment (SergeAstapov) [inline index.js]: @davideferre looks like I'm missing something. Wouldn't `indexOf('-prod') > -1` return `true` for both `ember b -prod` or `ember build --prod` commands?

Comment (jasonmit) [reply index.js]: process.argv is an array

Comment (SergeAstapov) [reply index.js]: Ah yes, my bad. Thank you!

Comment (jasonmit) [reply index.js]: Yeah it trolled me for a second as well :)

Comment (davideferre) [reply index.js]: It trolled me too ;)

**Pull request merged**